wemake_python_styleguide/visitors/ast/conditions.py

```diff
@@ -154,6 +154,16 @@ def _check_useless_len(self, node: AnyIf) -> None:
             if given_function_called(node.test, {'len'}):
                 self.add_violation(UselessLenCompareViolation(node))
 
+    def _has_else_expr(self, node: ast.If) -> bool:
```

Comment:
Let's move it to https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/logic/ifs.py